src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx

```diff
@@ -1,51 +1,82 @@
-  const [error, setError] = useState(false);
-  const onDrop = async (files: File[]): Promise<void> => {
-    try {
-      const file = files[0];
-      const config = await readFileAsJson<ConfigFile>(file);
-      setError(false);
-      onConfigFile(config);
-    } catch (e) {
-      setError(true);
-      console.error(e);
-    }
-  };
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = styled(
+  ({ className, onConfigFile, errorMessage }) => {
+    const [error, setError] = useState(false);
+    const onDrop = async (files: File[]): Promise<void> => {
+      try {
+        const file = files[0];
+        const config = await readFileAsJson<ConfigFile>(file);
+        setError(false);
+        onConfigFile(config);
+      } catch (e) {
+        setError(true);
+        console.error(e);
+      }
+    };
+    const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
 
-  return (
-    <>
-      <div className="py-3">
-        <h1>Upload Configuration File</h1>
-      </div>
-      {errorMessage && (
-        <div className="my-2">
-          <ErrorAlert message={errorMessage} />
-        </div>
-      )}
-      <div {...getRootProps()}>
-        <input data-testid="config-file-drop-zone" {...getInputProps()} />
-        <div
-          className={`text-center w-100 border-dashed border-2 border-gray-600 p-3 ${
-            isDragActive ? "bg-gray-400" : "bg-white"
-          }`}
-        >
-          {error && <div>Error: File cannot be read</div>}
-          <div>Drag and drop file here</div>
-          <div>or</div>
-          <div>Browse File</div>
-        </div>
+    return (
+      <div className={className}>
+        <Wrapper>
+          <ProgressBar step={1} />
+          <Title>Upload Configuration File</Title>
+          {errorMessage && (
+            <div className="my-2">
+              <ErrorAlert message={errorMessage} />
+            </div>
+          )}
+          <div {...getRootProps()}>
+            <input data-testid="config-file-drop-zone" {...getInputProps()} />
+            <div className={`dropZone ${isDragActive ? "bg-gray-400" : "bg-white"}`}>
+              {error && <div>Error: File cannot be read</div>}
+              <div className="dropText">Drag and drop file here</div>
+              <div className="or">or</div>
+              <ButtonSolidWhiteOrange className="browseButton">Browse File</ButtonSolidWhiteOrange>
+            </div>
+          </div>
+        </Wrapper>
       </div>
-    </>
-  );
-};
+    );
+  }
+)`
+  .dropZone {
+    border: 2px dashed #dddddd;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 62px;
+    padding-bottom: 52px;
+  }
+
+  .dropText {
+    font-weight: bold;
+    font-size: 18px;
+    color: ${vars.greyDark};
```

Comment:
for all font related matters, can use mixin `fontSize` or `fontRobotoRegular` directly

Reply:
ok